rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/internal/LRARecoveryModule.java

```diff
@@ -130,6 +135,35 @@ private void doRecoverTransaction(Uid recoverUid) {
         }
     }
 
+    private void moveEntryToFailedLRAPath (final Uid failedUid) throws ObjectStoreException {
+        String failedLRAType = LongRunningAction.getFailedLraType();
+        InputObjectState inputState = _recoveryStore.read_committed(failedUid, _transactionType);
+        InputObjectState failedLRAUidState = _recoveryStore.read_committed(failedUid, failedLRAType);
+        boolean result = false;
+
+        if (inputState != null) {
+            if (failedLRAUidState != null) {
+                // Record already exists in failedLRARecord_PATH, hence removing it from the LRA_PATH
+                _recoveryStore.remove_committed(failedUid, _transactionType);
```

Comment:
sure

Reply:
I was thinking rather the case when the `_recoveryStore.remove_committed` failed. Then it would be good to inform about the trouble during recovery processing.

When the removal was successful then a trace log could be fine as well (but that should not be `warnf` as it's now).

Reply:
ok